task-list/app/controllers/people_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PeopleController < ApplicationController
+
+  def index
+    @all_people = Person.all
+    render :index
```

Comment:
I don't think you need to say "render :index" here. Rails will do this automatically for the view that has the same name as the function you're in (like def index will render :index, and def edit will render :edit). You should remove it, since it makes it seem like you're doing something special/different than the standard.